src/advancedHMC_MCMC.jl

```diff
@@ -181,18 +184,79 @@ function physloglikelihood(Tar::LogTargetDensity, θ)
     for i in 1:length(Tar.prob.u0)
         # can add phystd[i] for u[i]
         physlogprob += logpdf(MvNormal(nnsol[i, :],
-                                       LinearAlgebra.Diagonal(map(abs2,
-                                                                  Tar.phystd[i] .*
-                                                                  ones(length(physsol[i, :]))))),
-                              physsol[i, :])
+                LinearAlgebra.Diagonal(map(abs2,
```

Comment:
It would be good to split this out into multiple lines, it's a bit unreadable right now

Reply:
done